tutorials/variational-quantum-eigensolver/vqe.ipynb

```diff
@@ -218,7 +218,7 @@
    "source": [
     "We can schedule a series of [`qiskit.transpiler`](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler) passes to optimize our circuit for a selected backend and make it compatible with the instruction set architecture (ISA) of the backend. This can be easily done using a preset pass manager from `qiskit.transpiler` and its `optimization_level` parameter.\n",
```

Comment:
```suggestion
    "Schedule a series of [`qiskit.transpiler`](https://docs.quantum-computing.ibm.com/api/qiskit/transpiler) passes to optimize the circuit for a selected backend and make it compatible its ISA. This can be easily done by using a preset pass manager from `qiskit.transpiler` and its `optimization_level` parameter.\n",
```